hal/armv7a/zynq7000/config.h

```diff
@@ -17,19 +17,23 @@
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
 
+#ifndef __ASSEMBLY__
+
 #include "zynq.h"
 #include "types.h"
 #include "peripherals.h"
 #include "../string.h"
 
-
-/* User interface */
 #define PATH_KERNEL "phoenix-armv7a9-zynq7000.elf"
 
-/* Addresses descriptions */
-#define ADDR_SYSPAGE   0xffff8000 /* Begin of OCRAM's high address */
-#define ADDR_BITSTREAM 0x00100000
+#endif
 
-#define SIZE_PAGE 0x1000
+
+#define SIZE_PAGE    0x1000
+#define SIZE_STACK   4 * SIZE_PAGE
+#define SIZE_SYSPAGE 5 * SIZE_PAGE
+
+#define ADDR_BITSTREAM 0x00100000
+#define ADDR_STACK     _end + SIZE_SYSPAGE + SIZE_STACK
```

Comment:
Use brackets for compound defines

Reply:
I've forgotten about making change it these files as well. Fixed.